plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties

```diff
@@ -450,6 +450,7 @@ copy_special_force_quote_cell_values_text = Always quote values
 copy_special_force_quote_cell_values_tip = Place all cell values in quotes
 copy_special_copy_as_html_text = Copy as HTML
 copy_special_copy_as_html_tip = Copy as HTML (in addition to plaintext format)
+copy_special_hint_for_hotkey = For future use with the same settings, you\nmay use the hotkey: {0}
```

Comment:
I would recommend to use
`The advanced copy settings are saved now, please use the key: {0} to invoke it`